backend/services/workflow.py

```diff
@@ -37,6 +38,10 @@ def __init__(
         )
 
         self.search = EmbeddingSearchService(tidb_config=tidb_config)
+        self.reranker = Reranker(
+            tidb_config=tidb_config, 
+            reranker_result_limit=self.settings.rerank_result_limit
+        )
```

Comment:
Use he re-ranker after we gather all the results from image embedding / text-embedding and text2sql